sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/_transport.py

```diff
@@ -794,6 +794,8 @@ def _read(self, n, initial=False, buffer=None, _errnos=None):  # pylint: disable
                 raise IOError("Websocket connection has already been closed.") from None
             except WebSocketTimeoutException as wte:
                 raise TimeoutError('Websocket receive timed out (%s)' % wte) from wte
+            except WebSocketConnectionClosedException as e:
```

Comment:
```suggestion
            except (WebSocketConnectionClosedException, SSLError) as e:
```

Reply:
Would an import be needed for SSLError? Are we seeing this pop up anywhere?